packages/server-wallet/package.json

```diff
@@ -8,6 +8,7 @@
     "@statechannels/client-api-schema": "^0.3.7",
     "@statechannels/nitro-protocol": "^0.3.7",
     "@statechannels/wallet-core": "^0.3.7",
+    "@statechannels/native-utils": "^0.1.4",
```

Comment:
Can you instead install `wasm-utils`?
- it appears like it will work on any (most?) systems
- `native-utils` does not install on my system